var/spack/repos/builtin/packages/py-awkward/package.py

```diff
@@ -25,12 +30,13 @@ class PyAwkward(PythonPackage):
     patch('pybind11.patch', when="@:1.2.2")
     patch('pybind11_02.patch', when="@1.2.3:")
 
-    depends_on('py-setuptools', type='build')
+    depends_on('py-setuptools@42.0:', type='build')
     depends_on('py-pyyaml', type='build')
 
     depends_on('python@2.7:2.8,3.5:', type=('build', 'run'))
     depends_on('py-numpy@1.13.1:', type=('build', 'run'))
     depends_on('py-pybind11', type=('build', 'link'))
     depends_on('dlpack', when="@1.0.0:")
     depends_on('rapidjson')
-    depends_on('cmake', type='build')
+    depends_on('cmake@3.13:', type='build')
+    depends_on('py-wheel@0.36.0:', type='build', when='@:1.7.0')
```

Comment:
Where does the wheel constraint come from?

Reply:
I checked the `pyproject.toml` file for other tags on github, and this was still present in 1.7.0:
https://github.com/scikit-hep/awkward-1.0/blob/952981a6f9851a55b41efec1ea64dbcd55698e6d/pyproject.toml#L4